tests/teal/sourcemap_compile_consistency/consistent.teal

```diff
@@ -0,0 +1,80 @@
+#pragma version 8
```

Comment:
feel free to delete this fixture and as well as the new unit tests in `compiler_test.py`. I've brought these into #682 in this commit: 174b9a6f7920ee3154ed57083ffe37bf0d63b73f

Reply:
👍 I will take it down then.

Reply:
I took expected teal and test related code in compiler down in [fddf4f6](https://github.com/algorand/pyteal/pull/684/commits/fddf4f63987d9173b0c01ae03434031a7b3d71d0)